docs/api-reference/sql-api.md

```diff
@@ -99,6 +103,23 @@ The request body takes the following properties:
     }
     ```
 
+##### Text Format Request body
+
+Druid also allows you to submit SQL queries in text format which is simpler than above JSON format. 
+To do this, just set the `Content-Type` request header to `text/plain` or `application/x-www-form-urlencoded`, and pass SQL via the HTTP Body. 
```

Comment:
Should mention that for `application/x-www-form-urlencoded`, the payload the user sends must be url-encoded.